dashboard/test/state/build_test.dart

```diff
@@ -309,8 +311,13 @@ void main() {
   });
 
   testWidgets('sign in functions call notify listener', (WidgetTester tester) async {
-    final MockGoogleSignInPlugin mockSignInPlugin = MockGoogleSignInPlugin();
-    when(mockSignInPlugin.onCurrentUserChanged).thenAnswer((_) => Stream<GoogleSignInAccount>.value(null));
+    final MockGoogleSignIn mockSignInPlugin = MockGoogleSignIn();
+    when(mockSignInPlugin.signIn()).thenAnswer((_) => Future<GoogleSignInAccount?>.value(null));
+    when(mockSignInPlugin.signOut()).thenAnswer((_) => Future<GoogleSignInAccount?>.value(null));
+    when(mockSignInPlugin.signInSilently()).thenAnswer((_) => Future<GoogleSignInAccount?>.value(null));
+    // FOR REVIEW:
+    // probably not the right way to stub null for signin and signout, but it passes the test
+    when(mockSignInPlugin.onCurrentUserChanged).thenAnswer((_) => Stream<GoogleSignInAccount?>.value(null));
```

Comment:
Stream.empty is useful for tests

https://api.dart.dev/stable/2.10.4/dart-async/Stream/Stream.empty.html